### PR TITLE
Client Reskin

### PR DESCRIPTION
**Pull request by kylekthompson**
| old | new |
| --- | --- |
| screen shot 2018-01-10 at 12 24 02 pm | screen shot 2018-01-24 at 1 49 30 pm |

### CODE REVIEW

File: client/src/components/Form/Errors.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { colors } from 'models/constants';
+import { fonts } from 'models/styles';
+
+const Icon = ({ className }) => (
```
Comment (atareshawty) [inline client/src/components/Form/Errors.jsx]: Maybe I'm missing something, but how can this `Icon` component get passed a `className` here?

File: client/src/models/validations/email.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { pluralize } from 'models/string';
+import Validator from 'models/validations/validator';
+
+const EMAIL_REGEX = /^.+@.+\..+$/;
```
Comment (atareshawty) [inline client/src/models/validations/email.js]: `kyle@kylekthompson.com@example.com` 😜

File: client/src/models/validations/validator.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default class Validator {
```
Comment (atareshawty) [inline client/src/models/validations/validator.js]: Are you using this class exclusively for form validation? If so, you could make it a bit more obvious with a better name. `FormValidator`?

File: client/src/components/SignInForm/state/handleInputBlur.js [hunk 4]
```diff
@@ -1,3 +1,26 @@
-const handleInputBlur = (state) => state;
+import { formValuesFromState } from 'models/form';
```
Comment (atareshawty) [inline client/src/components/SignInForm/state/handleInputBlur.js]: I like the idea of moving these handlers out of your component's file. It cleans things up a bit

File: client/src/components/Form/Errors.jsx (hunk 1 above)
Comment (kylekthompson) [reply client/src/components/Form/Errors.jsx]: It's passed by `styled-components`! (the `styled(Icon)` a little bit lower).

I've been meaning to abstract out the Icon components to take in a name that maps to the font awesome icon, but haven't gotten around to it. `ErrorIcon` is the actual one being used, which is the styled version of `Icon`

File: client/src/models/validations/email.js (hunk 2 above)
Comment (kylekthompson) [reply client/src/models/validations/email.js]:
```js
if (EMAIL_REGEX.exec(email) && occurenceCount(email, '@') === 1) { return null; }
```

Nice try!

File: client/src/models/validations/validator.js (hunk 3 above)
Comment (kylekthompson) [reply client/src/models/validations/validator.js]: Yep that's all it's being used for. I should probably also rename the folder to something form related

File: client/src/components/Form/Errors.jsx (hunk 1 above)
Comment (atareshawty) [reply client/src/components/Form/Errors.jsx]: Ah, I see

**Pull request merged**